Firestore/Source/Core/FSTFirestoreClient.h

```diff
@@ -48,6 +48,7 @@
 NS_ASSUME_NONNULL_BEGIN
 
 using firebase::firestore::api::DocumentSnapshot;
+using firebase::firestore::api::Settings;
```

Comment:
If you add the `namespace api = ...` I suggested above you don't need to add a using statement for each class.

Reply:
SGTM